package.json

```diff
@@ -35,7 +36,7 @@
     "coffee-script": "^1.12.5",
     "colors": "^1.1.2",
     "cross-spawn": "^5.0.1",
-    "dredd-transactions": "^4.3.0",
+    "dredd-transactions": "4.2.0",
```

Comment:
Sorry I thought I linked the issue. Currently the latest version of `dredd-transactions` has an issue introduced by one of its dependencies - https://github.com/apiaryio/dredd-transactions/issues/93